### PR TITLE
FEATURE: Taglines in Cards

### LINKED ISSUE #528: [FEATURE] Taglines in Cards
### Description

- Currently the cards has a description as a result, we have different height for different cards
- Make sure the description is not visible in the cards, just the **tagline** is visible
- Make sure all the cards has equal height, no cards must be greater than another

## Present change : 

image

## What we need : 

image

### Additional information

- You might need to get the data from Backend, so maybe write server side codes too 
- You need to get this done by like 7days (MAX)

### 🥦 Browser

Brave

### 👀 Have you checked for similar open issues?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/IAmTamal/Milan/blob/main/CONTRIBUTING.md)

### Are you willing to work on this issue ?

No, someone else can work on it

### PR DESCRIPTION
**Pull request by the-macson**
# Fixes Issue

**My PR closes #528 **

# Changes proposed

## What did you do ?

fixed tagline and box height 

---

# Check List (Check all the applicable boxes)

- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title and description of the PR is clear and explains the approach.

# Screenshots

Screenshot (278)

### CODE REVIEW

**Comment by the-macson** [comment]
@IAmTamal please review it

File: src/components/SingleClub/index.jsx [hunk 1]
```diff
@@ -43,10 +43,10 @@ export default function SingleClub({ club }) {
         </div>
 
         <div id={`less${club._id}`} className="desc">
-          {club.description.length > 80
-            ? club.description.slice(0, 80) + '...'
-            : club.description}
-          {club.description.length > 80 ? (
+          {club.tagLine.length > 40
```
Comment (IAmTamal) [inline src/components/SingleClub/index.jsx:46]: Can we remove this ? 
Like we cannot technically input a tagline with more than 50 characters. So i think maybe the expand feature is not of much use in this scenario

**Review by IAmTamal — CHANGES_REQUESTED** [review]
Your changes looks fantastic, however i just want to suggest a small change !

**Comment by the-macson** [comment]
Okay

File: src/components/SingleClub/index.jsx (hunk 1 above)
Comment (the-macson) [reply src/components/SingleClub/index.jsx]: changes done ✅✅✅✅✅

**Comment by the-macson** [comment]
changes done ✅✅✅✅✅

**Review by IAmTamal — APPROVED** [review]
## Looks good to me ✅

- **⚒ Thankyou for working on the issue**
- **⭐ Do star the repo, to get updates on more issues !**
- **💚 Keep contributing to Opensource**

**Pull request merged**